python/custreamz/custreamz/kafka.py

```diff
@@ -4,41 +4,84 @@
 
 import cudf
 
-from custreamz.utils import docutils
-
 
 # Base class for anything class that needs to interact with Apache Kafka
-class CudfKafkaClient(object):
+class CudfKafkaClient:
     def __init__(self, kafka_configs):
+
+        """
+        Base object for any client that wants to interact with a Kafka broker.
+        This object creates the underlying KafkaDatasource connection which
+        is used to read data from Kafka and create cudf Dataframes.
+        This class should not be directly instantiated.
+
+        Parameters
+        ----------
+        kafka_configs, dict, Key/Value pairs of librdkafka
+        configuration values. Full list of valid configuration
+        options can be found at
+        https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
```

Comment:
Need to correct formatting of docstrings, here's one for example:
```suggestion
        kafka_configs : dict
            Dict of key/value pairs of librdkafka configuration values. Full
            list of valid configuration options can be found at
            https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
```
first line is argument name, type, and whether it's optional or not
second+ lines are indented and are a description of the parameter